content/posts/networkx/GSoC-2022/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "GSoC 2022: NetworkX VF2++ Implementation"
+date: 2022-06-09 
+draft: false 
+description: "Getting started"
```

Comment:
Can you write something meaningful here? Sorry if I missed this one.

I am planning on merging this today if there are no more comments.

Reply:
Do you refer to the description?

Reply:
yes

Reply:
It's a tweet like description

Reply:
It's the first time i create a blog so i was not familiar with the philosophy. I promise that the next blogs to come will be a lot more complete and will not require such long time to review. :)

Reply:
No problem at all 😃. It's part of the learning and we are also learning our way here as we build this blog.

Reply:
Seems like you have some formatting issues. Cf CI. For next time, you can install pre-commit hooks to format automatically for you (I should update our guidelines to point this out)